consensus/src/round_manager.rs

```diff
@@ -1034,9 +1185,26 @@ impl RoundManager {
             .await
             .context("[RoundManager] Failed to process a newly aggregated QC");
         self.process_certificates().await?;
+        self.pending_order_votes
+            .garbage_collect(self.block_store.sync_info().highest_ordered_round());
```

Comment:
this is probably better in the new round event instead of here, this only covers cases the node aggregates the qc itself

Reply:
Done.